### PR TITLE
Redux branch

### PR DESCRIPTION
**Pull request by nayyyhaa**
## In this PR :
- I've migrated context to redux-toolkit.

#### files to *ignore*
- [ ] data files
- [ ] CSS files
- [ ] index files
- [ ] backend files

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
-  const { setShowFilterBar } = useSidebar();
   const location = useLocation();
-  const { auth } = useAuth();
+  // const { auth } = useAuth();
+  const { auth } = useSelector((store) => store.authReducer);
+  const { searchIp } = useSelector((store) => store.searchReducer);
+  const dispatch = useDispatch();
   const isSidebarVisible = !(location.pathname.includes("login") || location.pathname.includes("signup"));
-  const { searchIp, setSearchIp } = useSearch();
+  // const { searchIp, setSearchIp } = useSearch();
```
Comment (abhisheksahani-nova) [inline src/components/Navbar.jsx]: Remove comment if not needed

**Review by abhisheksahani-nova — APPROVED** [review]
Everything looks good and clean, nothing to suggest, good work.

**Review by vishal804 — APPROVED** [review]

**Comment by vishal804** [comment]
Code is readable and clean

File: src/components/Navbar.jsx (hunk 1 above)
Comment (nayyyhaa) [reply src/components/Navbar.jsx]: done

**Comment by nayyyhaa** [comment]
Thanks for the review, merging the changes. 👍

**Pull request merged**